examples/multi_storage_client/msc_config.yaml

```diff
@@ -0,0 +1,19 @@
+# This is an example MSC configuration file for accessing the cwb datasets stored 
+# in an S3-compatible bucket cwb-diffusions. 
+# The credentials are inferred from the environment variables S3_KEY and S3_SECRET.
+profiles:
+  cwb-diffusions:
+    storage_provider:
+      type: s3
+      options:
+        region_name: us-east-1
+        endpoint_url: https://pbss.s8k.io
```

Comment:
Remove this reference from here. A modulus example should be runnable by anyone without any specific network domain requirements. Please refactor to point this example to a publicly available zarr dataset. Here are a couple of dataset suggestions:

1. CMIP6 archive on AWS: https://registry.opendata.aws/cmip6/
2. ARCO ERA5 dataset on google cloud: https://github.com/google-research/arco-era5?tab=readme-ov-file#025-pressure-and-surface-level-data

Reply:
Thanks, I added examples of CMIP6.